config/environments/production.rb

```diff
@@ -41,10 +40,8 @@
   # config.action_dispatch.x_sendfile_header = 'X-Sendfile' # for Apache
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
-  # Mount Action Cable outside main process or domain
-  # config.action_cable.mount_path = nil
-  # config.action_cable.url = 'wss://example.com/cable'
-  # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
+  # Store uploaded files on the local file system (see config/storage.yml for options)
+  config.active_storage.service = :local
```

Comment:
I'm not sure it makes sense to have production set to `:local` even if we aren't using it right now. Maybe we can get away with just commenting it out.

Reply:
yeah, good catch. thanks!